inc/objectlock.class.php

```diff
@@ -173,7 +173,7 @@ function callUnlock( ) {
 
       $msg = "<table><tr><td class=red>";
       $msg .= __("Locked by you!")."</td>";
-      $msg .= "<td><a class='vsubmit' onclick='javascript:unlockIt(this);'>".__('Unlock ').
+      $msg .= "<td><a class='vsubmit' onclick='javascript:unlockIt(this);'>".__('Unlock ')."&nbsp;".
```

Comment:
'Unlock ' should be 'Unlock' and ' #' should be "\&nbsp; #" to preserve only one space between words?